src/model/eraser/EraseableStroke.cpp

```diff
@@ -33,9 +33,12 @@ void EraseableStroke::draw(cairo_t* cr) {
 
     double w = this->stroke->getWidth();
 
+    // Some tools ignore pressure sensitivity.
+    bool usePressure = this->stroke->getToolType() != STROKE_TOOL_HIGHLIGHTER;
+
     for (GList* l = tmpCopy->data; l != nullptr; l = l->next) {
         auto* part = static_cast<EraseableStrokePart*>(l->data);
-        if (part->getWidth() == Point::NO_PRESSURE) {
+        if (!usePressure || part->getWidth() == Point::NO_PRESSURE) {
```

Comment:
This should fix the eraser issue, but I'm not very satisfied with the solution -- I would prefer that EraseableStroke not need to take into account the type of tool that made the stroke, if possible.